examples/ota-requestor-app/linux/main.cpp

```diff
@@ -171,6 +179,12 @@ void ApplicationInit()
 {
     chip::Dnssd::Resolver::Instance().Init(chip::DeviceLayer::UDPEndPointManager());
 
+    if (gRequestorCanConsent.HasValue())
+    {
+        gRequestorCore.SetProviderParameters(gRequestorCanConsent.Value());
+        ChipLogProgress(SoftwareUpdate, "Setting RequestorCanConsent to %d", gRequestorCanConsent.Value());
```

Comment:
nit: Don't think we need this debug message. We can see it in the QueryImage command on the provider side